### PR TITLE
Sortby filters task

### PR DESCRIPTION
**Pull request by Ely-ullen**
Please note I know that tropics are not included in the filters  and they are hardcoded in another API , but I would like to move on to the next task so I can finish today and then update this later. 

Lmk :)

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -4,10 +4,15 @@ const axios = require("axios");
...
-const fetchArticlesData = () => {
+const fetchArticlesData = (sortBy, order) => {
   return axios
-    .get("https://eb-nc-news-app.herokuapp.com/api/articles")
-    .then((res) => res.data);
+    .get(`https://eb-nc-news-app.herokuapp.com/api/articles`, {
+      params: { sort_by: sortBy, order_by: order },
+    })
+    .then((res) => {
+      console.log(res.request.responseURL);
```
Comment (dappernerddesigns) [inline src/api.js]: Make sure to remove console logs you don't need

File: src/components/ArticlesHome.js [hunk 2]
```diff
@@ -7,22 +7,48 @@ import Moment from "moment";
...
+  };
+
   return (
     <>
+      <nav className="Filters">
+        <select class="select" value={sortBy} onChange={handleSortByChange}>
+          <option value="created_at">date added</option>
+          <option value="author">author</option>
+          <option value="title">title</option>
+          <option value="body">body</option>
```
Comment (dappernerddesigns) [inline src/components/ArticlesHome.js]: Why are you sorting on the body? What does it even mean?

File: src/components/ArticlesHome.js [hunk 3]
```diff
@@ -7,22 +7,48 @@ import Moment from "moment";
...
+          <option value="topic">topic</option>
```
Comment (dappernerddesigns) [inline src/components/ArticlesHome.js]: If you want to sort on topic later, remove it for now and come back at another time.

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
The sort functionality on all articles works very well, and is super quick. However when I click on any topic the sort functionality goes away. Currently sort only works for all articles. I would expect to see it anywhere I can view a list of articles.

**Comment by Ely-ullen** [comment]
Thank you! I will solve those issues

**Pull request merged**